app/src/main/java/tech/relaycorp/letro/account/model/Account.kt

```diff
@@ -8,12 +8,50 @@ const val TABLE_NAME_ACCOUNT = "account"
 
 @Entity(
     tableName = TABLE_NAME_ACCOUNT,
-    indices = [Index("veraId", unique = true)],
+    indices = [Index("veraidId", unique = true)],
 )
 data class Account(
     @PrimaryKey(autoGenerate = true)
     val id: Long = 0L,
-    val veraId: String,
+    val veraidId: String,
+    val requestedUserName: String,
+    val locale: String,
```

Comment:
Let's rename it to `normalisedLocale`? Because we only use this field after calling `normiliseString()` method